specification/purview/data-plane/Azure.Analytics.Purview.Catalog/preview/2023-02-01-preview/purviewcatalog.json

```diff
@@ -5674,6 +5774,17 @@
             "status": {
               "$ref": "#/definitions/TermStatus"
             },
+            "nickName": {
+              "type": "string",
+              "description": "The nick name of the term."
+            },
+            "hierarchyInfo": {
+              "type": "array",
+              "description": "The hierarchy information of the term.",
+              "items": {
+                "$ref": "#/definitions/PurviewObjectId"
+              }
+            },
```

Comment:
`AtlasGlossaryTerm` is used in API `/atlas/v2/glossary/term/{termGuid}`, so I assume it be atlas data type.

Now we add `nickName` and `hierarchyInfo` into it. Are they atlas data, or our data?

`PurviewObjectId` looks like our augmentation to `PurviewObjectId`.